production/helm/loki/values.yaml

```
@@ -680,6 +680,8 @@ enterprise:
     extraEnvFrom: []
     # -- The name of the PriorityClass for tokengen Pods
     priorityClassName: ""
+    # -- Use the host's user namespace in tokengen pods
+    hostUsers: true
```

```suggestion
    hostUsers: nil
```

The default should be nil, e.g. unset.

---

ok done